rfds/0000-establishing-the-rfd-process.md

```diff
@@ -0,0 +1,440 @@
+please contact one of the QWG Co-Chairs.
+
+Individuals wishing to submit an RFD who are not QWG participants should first
+request admittance to participate prior to submitting the RFD, so that they
+can be present for direct synchronous discussions about their proposal. If the
+scheduled time for synchronous QWG meetings is not workable for an RFD
+proposer then alternative QWG meeting times or special briefings to inform the
+QWG members may be discussed and approved by the QWG.
+
+RFDs submitted by individuals who are not QWG participants and have not
+requested participation in the QWG will be responded to with a prompt to
+participate in the QWG. If the submitter does not respond in a timely manner or
+declines to engage with the QWG, then the RFD will be closed and rejected,
+though the rejection will not preclude future consideration of the merits of
+the contents of the RFD if proposed at a future time consistent with the
+requirements of the RFD process.
+
+### What Should Require an RFD?
+
+Changes to the CVE Record Format of the following types will require an
+approved RFD prior to implementation:
+
+1. Adding, modifying, or removing fields
+2. Adding, modifying, or removing data constraints
+
+Changes to the format that do not do either of the above do not require an
+RFD, although RFDs could still be submitted to aid in discussion and tracking
+of a proposal. Other kinds of changes might include updating or adding
+descriptions to fields, or adding or modifying field examples.
+
+In general, if you are in doubt about whether a proposal should start with an
+RFD, ask the Quality Working Group.
+
+### How are RFDs submitted?
+
+RFDs should be submitted as Pull Requests against the [CVEProject/cve-schema
+Git repository on GitHub][gh_repo], made against the
+[`develop`][gh_repo_develop] branch. The precise steps for submitting a new
+RFD are:
+
+- [Fork the CVEProject/cve-schema repository][gh_forking] to your own user
+  account.
+- [Create a new branch for your RFD proposal][gh_branching], branching off of
+  the `develop` branch from CVEProject/cve-schema.
+- In this new branch, copy the file `rfds/_TEMPLATE.md` from the root of your
+  checked out copy of your fork of CVEProject/cve-schema to a new file of the
+  form `rfds/0000-<TITLE>.md`, replacing `<TITLE>` with an [ASCII][ascii] title
+  for the RFD, with words separated by dashes.
+- Complete the template as described within that copied document. The text in
+  each section of the template describes the substance of the material that
+  should be in that section.
+- [Commit the changes][gh_commit] with a descriptive commit message describing
+  the proposal.
+- [Open a pull request][gh_pr] from your feature branch to the `develop` branch on the
+  CVEProject/cve-schema repository.
+- After the pull request is open, amend the RFD proposal file to link to that
+  Pull Request. Leave the RFD number as `0000`, it will be modified upon
```

Comment:
Who is on the hook for this modification or what is the process intended to be for this?

Reply:
Generally I'd expect it to be done by one of the QWG chairs, whom I believe are also the people who have merge rights on the repo (and thus can use GitHub's "Maintainers can edit this branch" feature). They should be able to make a new commit on the PR's branch from the submitter's fork which amends the RFD number to the correct one before accepting the merge.

I can amend the proposal to make this explicit.

Reply:
Amended in 1e81089